services/integration/src/models/configOne.model.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const configOneSchema = new Schema({
+  projectId: {
+    type: String,
+    required: [true, 'ProjectId is a required field']
+  },
+  role: [
```

Comment:
this must be in plural: roles, since it is an array.

Reply:
en el push que se hizo despues , esta todo el cambio